src/main/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConfig.java

```diff
@@ -54,6 +55,7 @@
 		.define(DMSDK_TRANSFORM, Type.STRING, Importance.MEDIUM, "Name of a REST transform to use when writing documents")
 		.define(DMSDK_TRANSFORM_PARAMS, Type.STRING, Importance.MEDIUM, "Delimited set of transform names and values")
 		.define(DMSDK_TRANSFORM_PARAMS_DELIMITER, Type.STRING, ",", Importance.LOW, "Delimiter for transform parameter names and values; defaults to a comma")
+		.define(DOCUMENT_COLLECTIONS_ADD_TOPIC, Type.BOOLEAN, true,Importance.LOW, "Boolean that indicate if topics should be add to documents collections ")
```

Comment:
The README says the default value is "false", but this is "true" - should have a default value of "false"

Reply:
resolved